db/migrations/1575817780103-User.ts

```diff
@@ -4,13 +4,16 @@ export class User1575817780103 implements MigrationInterface {
   name = 'User1575817780103';
 
   public async up(queryRunner: QueryRunner): Promise<any> {
+    // Temporary chnages to schema just to prove google auth0 works
     await queryRunner.query(
       `CREATE TABLE "users" (
           "id" SERIAL NOT NULL,
           "first_name" character varying NOT NULL,
           "last_name" character varying NOT NULL,
           "email" character varying NOT NULL,
-          "password_digest" character varying NOT NULL,
+          "google_id" character varying DEFAULT NULL,
+          "google_picture" character varying DEFAULT NULL,
+          "password_digest" character varying,
```

Comment:
we shouldn't edit existing migrations - if you're changing a model, create a new migration.